docs/preview/03-Guidance/migrate-from-testing-framework.md

```diff
@@ -0,0 +1,175 @@
+# Migrate your test suite from Testing Framework to Arcus.Testing
+This guide will walk you through the process of migrating your test suite from using the Testing Framework to `Arcus.Testing`.
+
+## Replace `Codit.Testing.OutputComparison/Xslt` with `Arcus.Testing.Assert`
+The `Codit.Testing.OutputComparison` library has some functionality to compare different kinds of file types. The new `Arcus.Testing.Assert` library handles these comparisons from now on.
+
+Start by installing this library:
+```shell
+PM > Install-Package -Name Arcus.Testing.Assert
+```
+
+🔗 See the [feature documentation](../02-Features/assertion.md) for more info on the supported assertions.
+
+### XML
+You can use `AssertXml` like any other assertion method. Instead of returning a boolean and a message, it throws an exception with a detailed report in case of a difference.
+
+```diff
+- using Codit.Testing.OutputComparison;
++ using Arcus.Testing;
+
+string expectedXml = ...;
+string actualXml = ...;
+
+- using Stream expectedXmlStream = ...;
+- using Stream actualXmlStream = ...;
+- bool isEqual = Xml.Compare(
+-     actualXmlStream, expectedXmlStream, out string userMessage, nodesToIgnore: Array.Empty<string>());
+
++ AssertXml.Equal(expectedXml, actualXml);
+```
+
+Any nodes that should be ignored can be configured with passing additional options:
+
+```diff
+- using Codit.Testing.OutputComparison;
++ using Arcus.Testing;
+
+- bool isEqual = Xml.Compare(..., new[] { "ignore-this-node" });
++ AssertXml.Equal(..., options =>
++ {
++     options.AddIgnoreNode("ignore-this-node");
++ });
+```
+
+🔗 See the [feature documentation](../02-Features/assertion.md) for more info on the `AssertXml`.
+
+### JSON
+You can use `AssertJson` like any other assertion method. Instead of returning a boolean and a message, it throws an exception with a detailed report in case of a difference.
+
+```diff
+- using Codit.Testing.OutputComparison;
++ using Arcus.Testing;
+
+string expectedJson = ...;
+string actualJson = ...;
+
+- using Stream expectedJsonStream = ...;
+- using Stream actualJsonStream = ...;
+- bool isEqual = Json.Compare(
+-    actualJsonStream, expectedJsonStream, out string userMessage, nodesToIgnore: Array.Empty<string>());
+
++ AssertJson.Equal(expectedJson, actualJson);
+```
+
+Any nodes that should be ignored can be configured with passing additional options:
```

Comment:
"can be configured with passing"
I would say "_by_ passing"

Reply:
Yeah, is better, thx! Fixed that here: https://github.com/arcus-azure/arcus.testing/pull/109/commits/fe6ab4ad368aacde4b6935f5ebf5f86ad54982cb